x-pack/test/functional/config.js

```diff
@@ -119,6 +119,9 @@ export default async function ({ readConfigFile }) {
       logstashPipelines: {
         pathname: '/app/management/ingest/pipelines',
       },
+      casesStackManagement: {
```

Comment:
Could you please rename it to `cases`? We don't want to stick with the management name in variables because at some point cases will move from the management page.